deploy-manage/upgrade/orchestrator/upgrade-cloud-enterprise.md

```diff
@@ -37,7 +37,12 @@ The following table shows the recommended upgrade paths from older Elastic Cloud
 If you have to upgrade to any of the intermediate versions, follow the upgrade instructions of the relevant release before upgrading to 4.0.0:
 - [ECE 2.5 Upgrade](https://www.elastic.co/guide/en/cloud-enterprise/2.5/ece-upgrade.html)
 - [ECE 2.13 Upgrade](https://www.elastic.co/guide/en/cloud-enterprise/2.13/ece-upgrade.html)
-- [ECE 3.8 Upgrade](https://www.elastic.co/guide/en/cloud-enterprise/3.7/ece-upgrade.html)
+  
+  :::{note}
+  Upgrading to 2.13.0 is not recommended as it can cause issues and you may lose access to the admin console. We strongly recommend to upgrade to 2.13.4.
```

Comment:
```suggestion
  We don't recommend upgrading to 2.13.0, as it can cause issues and you may lose access to the admin console. We strongly recommend upgrading to 2.13.4.
```